core-java-modules/core-java-reflection-2/src/test/java/com/baeldung/reflection/access/privatemethods/InvokePrivateMethodsUnitTest.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.reflection.access.privatemethods;
+
+import com.google.common.primitives.Longs;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Method;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class InvokePrivateMethodsUnitTest {
+
+    private final long[] someLongArray = new long[] { 1L, 2L, 3L, 4L, 5L };
+
+    @Test
+    public void whenSearchingForLongValue_theCorrectIndexOfTheValueIsReturned() throws Exception {
```

Comment:
The method doesn't need to be `public`

Reply:
Same assumption here as I stated in reply to the previous comment.  👍